app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The associations between models must be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Which means 🤔 The Post belongs-to user through the `'author_id'` column of the Post table. Now, we need to show that, the  `'author_id'` reference  User class modal.
Your code should look like this:

```ruby
  # app/models/post.rb
  belongs_to :author, class_name: 'User'
```